metadeploy/api/views.py

```diff
@@ -338,11 +379,12 @@ def _prepare_org_serialization(org_id):
             }
         ).data
 
-    def list(self, request):
+    @extend_schema(
+        responses={200: {"type": "object", "additionalProperties": {"type": "object"}}}
+    )
+    def get(self, request):
```

Comment:
Changed to `get` because `list` results in a schema wrapped in `[]`. Also, since the keys returned here are arbitrary we can't annotate the response with an exact serializer, so the docs are pretty generic. We can improve this by returning something with known keys instead. Not sure if it's worth bothering the frontend for that though 🤷

```json
{
  "property1": {},
  "property2": {}
}
```